src/components/priceDisplay.js

```diff
@@ -9,7 +9,7 @@ export const PriceDisplay = () => {
   //Call necessary selectors for display data
   const dispatch = useDispatch();
   const quote = useSelector(quoteSelector);
-
+  const space = " ";
```

Comment:
This const can be removed, since it is not being used anymore (we went with the margin approach)

Reply:
Sure no problem,  I can remove it immediately after the merge or just do it now and commit again. Whichever you prefer.